pinot-common/src/main/java/com/linkedin/pinot/common/utils/TarGzCompressionUtils.java

```diff
@@ -50,6 +50,8 @@
 public class TarGzCompressionUtils {
   private static final Logger LOGGER = LoggerFactory.getLogger(TarGzCompressionUtils.class);
   public static final String TAR_GZ_FILE_EXTENTION = ".tar.gz";
+  public static final String EXTRACT_FILE_OUTSIDE_OF_TARGET_DIR =
```

Comment:
don't have to make this public just for testing purpose. just test for the exception type.